src/components/auth-code/auth-code-service.ts

```diff
@@ -7,25 +7,38 @@ import {
   Http,
 } from "../../utils/http";
 import { AuthCodeResponse, AuthCodeServiceInterface } from "./types";
-import { getApiBaseUrl } from "../../config";
+import { getApiBaseUrl, supportAuthOrchSplit } from "../../config";
 
 export function authCodeService(axios: Http = http): AuthCodeServiceInterface {
   const getAuthCode = async function (
     sessionId: string,
     clientSessionId: string,
     sourceIp: string,
-    persistentSessionId: string
+    persistentSessionId: string,
+    clientSession: UserSessionClient
   ): Promise<ApiResponseResult<AuthCodeResponse>> {
-    const response = await axios.client.get(
-      API_ENDPOINTS.AUTH_CODE,
-      getRequestConfig({
-        baseURL: getApiBaseUrl(),
-        sessionId: sessionId,
-        clientSessionId: clientSessionId,
-        sourceIp: sourceIp,
-        persistentSessionId: persistentSessionId,
-      })
-    );
+    const config = getRequestConfig({
+      baseURL: getApiBaseUrl(),
```

Comment:
I thought that a difference between pre and post split is that the current Auth Code lambda is part of the OIDC API whereas the new one is part of the Frontend (Internal) API. So then the base URL would be different. Do you agree?

Reply:
Ooooh I'd missed that as a nuance, but agreed.

Reply:
Will [this environment variable](https://github.com/alphagov/di-authentication-frontend/blob/a903f81fac8efb4f00d1815ea10d35fee4798ca4/src/config.ts#L14) be pointing to the right place, or do we need a new one?

Reply:
That's the one, yeah 👍

Reply:
Thanks, updated